dbt/models/spatial/spatial.stadium.sql

```diff
@@ -2,9 +2,29 @@
     config(materialized='table') 
 }}
 
-SELECT
-    stadium.name,
-    ST_ASBINARY(ST_POINT(stadium.lon, stadium.lat)) AS geometry,
-    ST_ASBINARY(ST_POINT(stadium.x_3435, stadium.y_3435)) AS geometry_3435,
-    CAST(stadium.year AS VARCHAR) AS year
-FROM {{ ref('spatial.stadium_raw') }} AS stadium
+WITH distinct_years AS (
+    SELECT DISTINCT CAST(year AS INTEGER) AS year
+    FROM {{ ref('spatial.stadium_raw') }}
+),
```

Comment:
**issue (blocking):** This will only create records for years which exist in the original seed data, in the current case 2000, 2006, and 2017. You should amend this to create a record for all years between the min year present in the seed data and the max year of parcel data. See this [SO answer](https://stackoverflow.com/a/34994136) for a possible way to do this.